Lib/ntpath.py

```diff
@@ -23,6 +23,28 @@
 import genericpath
 from genericpath import *
 
+
+try:
+    import _winapi
+
+    def _normcase(s):
+        is_bytes = isinstance(s, bytes)
+        if is_bytes:
+            s = os.fsdecode(s)
+        s = _winapi.LCMapStringEx(_winapi.LOCALE_NAME_INVARIANT,
+                                  _winapi.LCMAP_LOWERCASE,
+                                  s.replace('/', '\\'))
+        if is_bytes:
+            s = os.fsencode(s)
+        return s
+except ImportError:
+    def _normcase(s):
+        if isinstance(s, bytes):
+            return s.replace(b'/', b'\\').lower()
+        return s.replace('/', '\\').lower()
+
+
+
```

Comment:
This should be defined near the definition `normcase()`. Maybe follow the pattern of `normpath()` by using a `from` import, and just define `normcase()` itself twice. For example:

```python
try:
    from _winapi import (
        LCMapStringEx as _LCMapStringEx,
        LOCALE_NAME_INVARIANT as _LOCALE_NAME_INVARIANT,
        LCMAP_LOWERCASE as _LCMAP_LOWERCASE)

    def normcase(s):
        """Normalize case of pathname.
        
        Makes all characters lowercase and all slashes into backslashes.
        """
        if isinstance(s, bytes):
            return os.fsencode(
                        _LCMapStringEx(_LOCALE_NAME_INVARIANT,
                                       _LCMAP_LOWERCASE,
                                       os.fsdecode(s).replace('/', '\\')))
        else:
            return _LCMapStringEx(_LOCALE_NAME_INVARIANT,
                                  _LCMAP_LOWERCASE,
                                  s.replace('/', '\\'))
except ImportError:
    def normcase(s):
        """Normalize case of pathname.
        
        Makes all characters lowercase and all slashes into backslashes.
        """
        if isinstance(s, bytes):
            return os.fsencode(os.fsdecode(s).replace('/', '\\').lower())
        return s.replace('/', '\\').lower()
```
Note that the first example I gave forgot to update the fallback function to use `fsdecode()` for `bytes`. I corrected that in the above example. Also, the final triple quote in a multi-line docstring is supposed to go on its own line.